roxctl/common/flags/config_file.go

```diff
@@ -0,0 +1,139 @@
+package flags
+
+import (
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/logging"
+	"os"
+
+	"github.com/stackrox/rox/pkg/pointers"
+)
```

Comment:
Please fix the order of imports and check that all of them are used and none are missing.

```suggestion
import (
	"os"

	"github.com/ghodss/yaml"
	"github.com/pkg/errors"
	"github.com/spf13/cobra"
	"github.com/stackrox/rox/pkg/env"
	"github.com/stackrox/rox/pkg/logging"
	"github.com/stackrox/rox/pkg/pointers"
)
```